core/src/main/java/io/grpc/ExperimentalApi.java

```diff
@@ -35,7 +35,8 @@
  * </ol>
  *
  * <p>Note: This annotation is intended only for gRPC library code. Users should not attach this
- * annotation to their own code.
+ * annotation to their own code. If you want to check APIs' usage, you can use
+ * grpc-java-api-checker.
```

Comment:
How about:

```
 * @see <a href="https://github.com/grpc/grpc/grpc-java-api-checker">grpc-java-api-checker</a>, an 
 * Error Prone plugin to automatically check for usages of this API.
```

(Same for Internal.java)

Reply:
Done.